amp_conf/htdocs/admin/libraries/modulefunctions.class.php

```diff
@@ -2964,7 +2964,7 @@ function url_get_contents($url,$request,$verb='get',$params=array()) {
 		if(!$amp_conf['MODULEADMINWGET']) {
 			$pest = FreePBX::Curl()->pest($url);
 			try{
-				$contents = $pest->$verb($url.$request,$params);
+				$contents = $pest->$verb($url.$request);
```

Comment:
Opps I didn't mean to include that particular patch I meant to raise it as a bug separately. Willm update on the weekend.